homeassistant/components/flo/device.py

```diff
@@ -35,6 +35,7 @@ def __init__(
             name=f"{FLO_DOMAIN}-{device_id}",
             update_interval=timedelta(seconds=60),
         )
+        self._failure_count: int = 0
```

Comment:
Can be set outside of the constructor

Reply:
?

Reply:
are you saying to make this a class var? if so, for what reason?